cves/CVE-2012-5154.yml

```
@@ -32,7 +32,11 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  New:
+  A common vulnerability, Integer overflow, in a web browser could allow attackers to
```

Fix the wording, no need to cite that it's a common vulnerability.
```suggestion
  An Integer overflow in a web browser could allow attackers to
```